modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -3,58 +3,22 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```

Comment:
Edited green changes only; please resubmit if you would like the white and blue edited as well.